pkg/infra/usagestats/usage_stats.go

```diff
@@ -151,7 +151,11 @@ func (uss *UsageStatsService) sendUsageStats(oauthProviders map[string]bool) {
 	data := bytes.NewBuffer(out)
 
 	client := http.Client{Timeout: 5 * time.Second}
-	go client.Post(usageStatsURL, "application/json", data)
+	go func() {
+		if _, err := client.Post(usageStatsURL, "application/json", data); err != nil {
+			metricsLogger.Error("Failed to post stats", "usageStatsURL", usageStatsURL, "err", err)
```

Comment:
Don't see the point of including `usageStatsURL` since it's a constant.

```suggestion
			metricsLogger.Error("Failed to send usage stats", "err", err)
```